superset-frontend/plugins/legacy-preset-chart-deckgl/src/utilities/controls.jsx

```diff
@@ -37,6 +37,8 @@ export const D3_FORMAT_OPTIONS = [
   [',.1%', ',.1% (12345.432 => 1,234,543.2%)'],
   ['.3%', '.3% (12345.432 => 1234543.200%)'],
   ['.4r', '.4r (12345.432 => 12350)'],
+  [',.1f', ',.1f (12345.432 => 12,345.4)'],
```

Comment:
﻿﻿﻿I remember that users can enter d3format manually. Why do we need to create a format in deckgl that is not consistent with all other charts?

<img width="1115" alt="image" src="https://user-images.githubusercontent.com/2016594/155721403-170e5cc7-b1a9-4c9f-9714-b9896291b13e.png">